src/algebraic_geometry/prime_spectrum/basic.lean

```diff
@@ -466,6 +466,57 @@ lemma comap_singleton_is_closed_of_is_integral (f : R →+* S) (hf : f.is_integr
 (is_closed_singleton_iff_is_maximal _).2 (ideal.is_maximal_comap_of_is_integral_of_is_maximal'
   f hf x.as_ideal $ (is_closed_singleton_iff_is_maximal x).1 hx)
 
+lemma localization_comap_inducing (f : R →+* S) (M : submonoid R)
+  [@@is_localization _ M S _ f.to_algebra] : inducing (comap f) :=
```

Comment:
@Vierkantor Is this the best way to say that a ring hom is a localization?
Should we introduce a predicate `ring_hom.is_localization`?

Reply:
For the moment, this is the best way. I guess defining `ring_hom.is_localization` would work, but depending on the goals of this PR, we could also try:
```suggestion
  [algebra R S] [is_localization M S] : inducing (comap (algebra_map R S)) :=
```

Reply:
@erdOne what do you think?